boston-crime-forecasting/content.md

```diff
@@ -4,11 +4,27 @@ a Boston weather dataset and try to forecast crimes per hour.
 
 
 ## Set up dependencies
-Use Anaconda to install
-* `conda install "python=3.6.6" "scipy=1.1" "h5py<3" "tensorflow=1.14" "Keras=2.2" "scikit-learn=0.20"`
+Download Anaconda if you dont have it.
 
-* `pip install mlflow[extras]==1.2.0 "mflux-ai>=0.4.0"`
+Make a folder called `boston_crime` and in it a file called `environment.yml`. These are the dependencis 
+we will require, paste this into the environment file:
+```python
+name: boston2
+dependencies:
+  - python=3.6.6
+  - numpy=1.16.4
+  - tensorflow-gpu
+  - matplotlib=3.1.1
+  - pandas=0.25.1
+  - scikit-learn=0.21
+  - keras=2.2.4
+  - h5py<3
+```
+
+Open an Anaconda prompt, navigate to the `boston_crime` folder and write `conda env create -f environment.yml`. Now activate 
+this environment with `conda activate boston`.
 
+Then write `pip install mlflow[extras]==1.2.0 "mflux-ai>=0.4.0"`.
```

Comment:
It would be even smoother to include this in the environment.yml file, like this:
```
(...)
  - keras=2.2.4
  - h5py<3
  - pip:
    - mlflow[extras]==1.2.0
    - mflux-ai>=0.4.0
```